<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNewDataTransfer</class>
 <widget class="QDialog" name="QNewDataTransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>属性</string>
   </property>
   <widget class="QRadioButton" name="Timed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>计时</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Triggered">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>54</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>触发</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Interval_Time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>18</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>21</y>
      <width>22</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>秒</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Trigger_Bit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>52</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LB0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Trigger_Bit_Keyboard">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>20</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>111</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>地址类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Address_Type">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>位</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>字</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>双字</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>字/位数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Word_Bit_Number">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>128</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="bAutomatismOff">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>自动复位</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_length">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>动态长度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="length_Address">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>161</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LW0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_length">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>159</y>
      <width>20</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>源 地 址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Source_Address">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>11</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LB0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Source_Address_Keyboard">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>9</y>
      <width>20</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Destination_Address_Keyboard">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>39</y>
      <width>20</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Destination_Address">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>41</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LB0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>目标地址：</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Accept">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Timed</tabstop>
  <tabstop>Interval_Time</tabstop>
  <tabstop>Triggered</tabstop>
  <tabstop>Trigger_Bit</tabstop>
  <tabstop>Trigger_Bit_Keyboard</tabstop>
  <tabstop>bAutomatismOff</tabstop>
  <tabstop>Address_Type</tabstop>
  <tabstop>Word_Bit_Number</tabstop>
  <tabstop>Source_Address</tabstop>
  <tabstop>Source_Address_Keyboard</tabstop>
  <tabstop>Destination_Address</tabstop>
  <tabstop>Destination_Address_Keyboard</tabstop>
  <tabstop>Accept</tabstop>
  <tabstop>cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
